tsl/src/bgw_policy/reorder_api.c

```diff
@@ -245,31 +244,30 @@ policy_reorder_remove(PG_FUNCTION_ARGS)
 {
 	Oid hypertable_oid = PG_GETARG_OID(0);
 	bool if_exists = PG_GETARG_BOOL(1);
+	Hypertable *ht;
+	Cache *hcache;
 
 	PreventCommandIfReadOnly("remove_reorder_policy()");
 
-	int ht_id = ts_hypertable_relid_to_id(hypertable_oid);
+	ht = ts_hypertable_cache_get_cache_and_entry(hypertable_oid, CACHE_FLAG_NONE, &hcache);
 
 	List *jobs = ts_bgw_job_find_by_proc_and_hypertable_id(POLICY_REORDER_PROC_NAME,
 														   INTERNAL_SCHEMA_NAME,
-														   ht_id);
+														   ht->fd.id);
+	ts_cache_release(hcache);
```

Comment:
Same here. Seems like you only increase the risk of making a mistake when maintaining the code.

Reply:
Explained above.